pom.xml

```diff
@@ -5,7 +5,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>dev.cdevents</groupId>
 	<artifactId>cdevents-sdk-java</artifactId>
-	<version>0.1.0-SNAPSHOT</version>

```

Comment:
Updated this version to 0.1.2 as currently the SDK follows v0.1.2 spec

Reply:
Thanks! 
As I understand it, this will then be updated to `0.1.2` (without the `-SNAPSHOT`) when we make a release, and for now `0.1.2-SNAPSHOT` is fine.  
@aalmiray is that correct?

Reply:
Yes, that is correct. The convention is to keep `-SNAPSHOT` while doing development. This tag must go away during release.